test/e2e-tests-yaml.sh

```diff
@@ -29,21 +29,15 @@ header "Setting up environment"
 set +o errexit
 set +o pipefail
 
-echo ">> Deploying Pipeline CRD"
-ko apply -f config/ || fail_test "Build pipeline installation failed"
-
-for res in pipelineresources tasks pipelines taskruns pipelineruns; do
-  kubectl delete --ignore-not-found=true ${res}.pipeline.knative.dev --all
-done
+install_pipeline_crd || fail_test "Pipeline CRD head release installation failed"
```

Comment:
You're already failing the test in the function, `fail_test` will never be executed.
```suggestion
install_pipeline_crd
```